realm/realm-jni/project.mk

```diff
@@ -38,7 +38,9 @@ endif
 #endif
 
 ifeq ($(REALM_ANDROID),)
-  REALM_LDFLAGS += -llog
+  ifneq ($(OS),Darwin)
```

Comment:
I feel that we should create a `REALM_ROBOLECTRIC` to indicate it is compiling for the robolectric. And check for the `REALM_ROBOLECTRIC` here in case we support win&linux later.